pkg/helm/release/manager.go

```diff
@@ -352,17 +352,14 @@ func createJSONMergePatch(existingJSON, expectedJSON []byte) ([]byte, error) {
 // UninstallRelease performs a Helm release uninstall.
 func (m manager) UninstallRelease(ctx context.Context, opts ...UninstallOption) (*rpb.Release, error) {
 	// Get history of this release
-	h, err := m.storageBackend.History(m.releaseName)
+	_, err := m.storageBackend.History(m.releaseName)
 	if err != nil {
+		if errors.Is(err, driver.ErrReleaseNotFound) {
+			return nil, err
+		}
 		return nil, fmt.Errorf("failed to get release history: %w", err)
```

Comment:
I don't think this should be necessary since we're wrapping the error and using [`errors.Is()`](https://github.com/operator-framework/operator-sdk/blob/5c77be39bbdae146b0a18f550a6c1af4f26e3eaf/pkg/helm/controller/reconcile.go#L106-L119) on the calling side to see if it was a `driver.ErrReleaseNotFound` error.